test_pages/codein.withgoogle.com-Comment-2015-12-12.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>How GCI Website Display Links for Privly</title>
+
+  <style>
+    body { margin: 10px; padding: 0px; font-family: Arial; }
+  </style>
+</head>
+<body>
+  
+  <!-- All information about test case are added to the information_about_test_case -->
+  <div id="information_about_test_case">
```

Comment:
You don't need to include any of this information. I plan on opening a GCI task to integrate this page with our continuous integration system, TravisCI. Basically a computer will repeatedly visit this page and tell us if the extension is working for this page. People will rarely see it and we are going to capture many test cases just like this.